Makefile

```diff
@@ -1,21 +1,26 @@
+ifdef ACCOUNT
+else
+        ACCOUNT := 442294194136
+        export
+endif
 
-BUILD_SUBDIRS := lib awsconfig cloudtrail awsconfigrules awsconfig-notification-alert enhancesnapshot
```

Comment:
lib should be packaged with the particle, specially if the particle is deploying a lambda. Does removing this still package lib folder?

Reply:
no, it will not package lib folder. if you want to package it, include it in mapping file.

Reply:
The changes seems to only deploy managed-vpc and managed-os particles, none of which requires lib folder. This should be good.

Reply:
In Managed OS we deploy GetAMIId Lambda which requires lib folder.

Reply:
No, it doesn't need lib folder.